TelCoColorCoder_run.cpp

```diff
@@ -0,0 +1,40 @@
+#include "ColorCode_enum.hpp"
+#include "TelCoColorCoder.hpp"
+
+void testNumberToPair(int pairNumber, MajorColor expectedMajor,MinorColor expectedMinor)
```

Comment:
Production code needs to be kept separately from test code so that test code can be excluded during shipment of software to field.
Separation helps in maintenance and most importantly helps in testing production code independently and effectively .